internal/common/common.go

```diff
@@ -150,8 +151,25 @@ func GetNetworkCidrAttr(obj interface{}, fieldName string) []*string {
 	}
 
 	funk.ForEach(field, func(elem interface{}) {
-		addresses = append(addresses, swag.String(string(funk.Get(elem, "Cidr").(models.Subnet))))
+		address := funk.Get(elem, "Cidr")
+		if address != nil {
```

Comment:
This is a bugfix I'm piggybacking here. Without this check it could happen that we try to get `Cidr` from an object without `Cidr` and then cast it to `models.Subnet`. Runtime complains then that we try to convert interfaces in an illegal way.